File: app/src/main/java/com/example/sandsyndicate/Admin/MainActivity.java
package com.example.sandsyndicate.Admin;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.provider.ContactsContract;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

import com.example.sandsyndicate.R;

public class MainActivity extends AppCompatActivity {
    Button b1;
    EditText ed1,ed2;
    @Override
    protected void onCreate(Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);
        b1=(Button) findViewById((R.id.Login));
        ed1=(EditText) findViewById(R.id.USER);
        ed2=(EditText) findViewById(R.id.PASSWORD);
        b1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                if(ed1.getText().toString().isEmpty())

                {

                }
            }
        }
    }
}

File: app/src/main/java/com/example/sandsyndicate/MainActivity.kt
package com.example.sandsyndicate

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import android.os.Looper
import android.view.animation.Animation
import android.view.animation.AnimationUtils
import com.example.sandsyndicate.Admin.AdminHome
import com.example.sandsyndicate.Authentication.Login
import com.example.sandsyndicate.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        Handler().postDelayed({
            startActivity(Intent(this,Login::class.java))
        },1000)
        val animationf=AnimationUtils.loadAnimation(this,R.anim.fade)
        binding.mainbutton.startAnimation(animationf);
    }
}
